src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -96,25 +97,70 @@ public static AminoAcid getEukaryoticAminoAcidByCodon(final String codon) {
     }
 
     /**
-     * Returns the {@link AminoAcid} corresponding to the given three-letter Mitochondrial {@code codon}.
+     * Returns the {@link AminoAcid} corresponding to the given three-letter Mitochondrial {@code rawCodon}.
      * The codons given are expected to be valid for Mitochondrial DNA.
-     * @param codon The three-letter codon (each letter one of A,T,G,C) representing a Mitochondrial {@link AminoAcid}
-     * @return The {@link AminoAcid} corresponding to the given {@code codon}.  Returns {@code null} if the given {@code codon} does not code for a Mitochondrial {@link AminoAcid}.
+     * Assumes no special cases for alternate initiation sites.
+     * @param rawCodon The three-letter codon (each letter one of A,[T or U],G,C) representing a Mitochondrial {@link AminoAcid}
+     * @return The {@link AminoAcid} corresponding to the given {@code rawCodon}.  Returns {@code null} if the given {@code rawCodon} does not code for a Mitochondrial {@link AminoAcid}.
      */
-    public static AminoAcid getMitochondrialAminoAcidByCodon(final String codon, final boolean isFirst) {
+    public static AminoAcid getMitochondrialAminoAcidByCodon(final String rawCodon) {
```

Comment:
Is null okay here?  This should be documented and checked.  Same goes for all public methods.

Reply:
No.  Null is not allowed.  Since this method aliases another I have left the check in there and updated the comments here.